concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -999,3 +999,116 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_task_
         self.assertEqual(message_from_concent.file_transfer_token.operation, FileTransferToken.Operation.download)
 
         self._assert_client_count_is_equal(2)
+
+    def test_concent_requests_task_result_from_provider_when_compute_task_def_deadline_is_float(self):
+        """
+        Tests if on requestor ForceGetTaskResult message Concent will return ForceGetTaskResultUpload
+        if provider uploads correct result, but compute_task_def['deadline'] is given as float type.
+
+        Expected message exchange:
+        Requestor -> Concent:     ForceGetTaskResult
+        Concent   -> Requestor:   AckForceGetTaskResult
+        Concent   -> Provider:    ForceGetTaskResultUpload
+        """
+
+        # STEP 1: Requestor forces get task result via Concent.
+        # Concent accepts request if all conditions were met.
+        compute_task_def = self._get_deserialized_compute_task_def(
+            task_id='99',
+            subtask_id='8',
+            deadline="2017-12-01 11:00:00"
+        )
+        compute_task_def['deadline'] = 1512126000.999999
+        deserialized_task_to_compute = self._get_deserialized_task_to_compute(
+            compute_task_def=compute_task_def,
+            deadline="2017-12-01 11:00:00"
+        )
+        deserialized_report_computed_task = self._get_deserialized_report_computed_task(
+            task_to_compute=deserialized_task_to_compute,
+            timestamp="2017-12-01 11:00:01",
+        )
+        serialized_force_get_task_result = self._get_serialized_force_get_task_result(
+            report_computed_task=deserialized_report_computed_task,
+            timestamp="2017-12-01 11:00:01",
+        )
+        deserialized_force_get_task_result = load(
+            serialized_force_get_task_result,
+            CONCENT_PRIVATE_KEY,
+            self.REQUESTOR_PUBLIC_KEY,
+            check_time=False,
+        )
+
+        with freeze_time("2017-12-01 11:00:01"):
+            response_1 = self.client.post(
+                reverse('core:send'),
+                data=serialized_force_get_task_result,
+                content_type='application/octet-stream',
+            )
+
+        self.assertEqual(response_1.status_code, 200)
+        self._assert_stored_message_counter_increased(increased_by = 2)
```

Comment:
It does not come along with our new coding style - should be: `self._assert_stored_message_counter_increased(increased_by=2)`

Reply:
Test deleted